pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -1138,7 +1137,6 @@ section.quote {
       cursor: pointer;
       white-space: normal;
       height: -webkit-fit-content;
-      flex: 1;
```

Comment:
Can you see how this looks translated (Spanish and/or French)? I think this was preventing a UI bug if the tab label goes to two lines.

Reply:
I think it looks okay. Spanish and German definitely did; French is slightly off but is also off on production.

Production:

![Screenshot 2024-07-22 at 12 43 33 PM](https://github.com/user-attachments/assets/4e3b4e51-6cf4-4e40-bdf8-f209e8bdcdb2)

Local:

![Screenshot 2024-07-22 at 12 50 59 PM](https://github.com/user-attachments/assets/2c9e7dbb-1290-4614-ae28-2060cac2526a)


For context, I removed this line so that the end of the content aligned with the end of the tabs. It is a really small issue if you think removing it will cause issues.

Reply:
Oh that makes sense! This is fine to remove 👍